components/FeedItemCard.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <v-hover>
+    <template v-slot="{ isHovering, props }">
+      <v-card v-bind="props">
+        <v-img :src="src()" aspect-ratio="1">
+          <template #placeholder>
+            <div class="d-flex align-center justify-center fill-height">
+              <v-container fluid class="fill-height">
+                <v-row class="ma-0">
+                  <v-col
+                    cols="12"
+                    class="pending font-weight-md-thin scale-text text-center"
+                  >
+                    {{ id }}
+                  </v-col>
+                  <v-col cols="12">
+                    <v-progress-linear
+                      indeterminate
+                      color="black"
+                      background-color="transparent"
+                      height="1"
+                      bottom
+                    />
+                  </v-col>
+                </v-row>
+              </v-container>
+            </div>
+          </template>
+          <template #error>
+            <v-container fluid class="fill-height">
+              <v-row class="ma-0">
+                <v-col cols="12">
+                  <div class="error404 scale-text text-center">
+                    404 image not found
+                  </div>
+                </v-col>
+              </v-row>
+            </v-container>
+          </template>
+          <template #default>
+            <v-fade-transition>
+              <div v-if="isHovering" class="hover fill-height pa-1 pl-4">
+                <div v-if="isPlayable()" id="playIcon">
+                  <v-icon>{{ 'mdi-play' }}</v-icon>
+                </div>
+                <v-row align="end" class="fill-height pa-1 pl-4">
+                  <v-col>
+                    <a class="white--text font-weight-bold">
+                      {{ data?.title || "Cool Artwork Piece" }}
+                    </a>
+                    <br>
+                    <a class="white--text font-italic">
+                      {{ data?.creator || "Cool Artist" }} 
```

Comment:
you don't really need placeholders since we don't want to show hover overlay if there's a 404 or error